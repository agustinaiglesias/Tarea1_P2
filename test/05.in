definirFecha 28/02/2020
siguienteDia 
fechaActual
fechaActualExtendida
Fin