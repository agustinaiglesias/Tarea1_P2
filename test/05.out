1>Fecha definida en forma exitosa.
2>Se aumentó un día.
3>La fecha actual es 29/2/2020.
4>La fecha actual es 29 de Febrero de 2020.
5>Fin.
